src/util/sys/mod.rs

```diff
@@ -0,0 +1,32 @@
+#[cfg(target_os = "linux")]
+use libc;
+#[cfg(target_os = "linux")]
+use std::io::Error;
+
+pub enum AdjustPriType {
+    Incr,
+    Normal,
+    Desc,
+}
+
+#[cfg(target_os = "linux")]
+pub fn adjust_priority(t: AdjustPriType) {
+    unsafe {
+        let pid = libc::pthread_self();
+        let pri = match t {
+            AdjustPriType::Incr => -1,
+            AdjustPriType::Normal => 0,
+            AdjustPriType::Desc => 1,
+        };
+        if libc::setpriority(libc::PRIO_PROCESS as u32, pid as u32, pri) != 0 {
```

Comment:
the setpriority can support the nice value like in range [-20, 19], see https://linux.die.net/man/2/setpriority, why do we only have -1, 0 and 1 here? 

How can we set different nice values for other threads?

Reply:
Do we have such demand? I just don't want to make it too complicated.